### PR TITLE
Update script http 046

### PR DESCRIPTION
**Pull request by istoimenova**

### CODE REVIEW

**Comment by istoimenova** [comment]
@dboltovskyi, please review.

File: test_scripts/Polices/build_options/046_ATF_P_TC_PTU_UTF8_Encoding_Check_HTTP.lua [hunk 1]
```diff
@@ -114,12 +114,19 @@ require("user_modules/AppTypes")
...
+      if(d.payload.requestType == "HTTP") then
+        request_http_received = true
+        ptu = json.decode(d.binaryData)
+      end
     end)
-  :Times(AtLeast(1))
-  :Pin()
+  :Times(2)
+
+  if(request_http_received == false) then
```
Comment (dboltovskyi) [inline test_scripts/Polices/build_options/046_ATF_P_TC_PTU_UTF8_Encoding_Check_HTTP.lua]: @istoimenova, Due to asynchronous registering of expectations (in ATF) test case will be failed before 2nd OnSystemRequest received
One of the solutions is to implement verification in other test step

Comment (istoimenova) [inline test_scripts/Polices/build_options/046_ATF_P_TC_PTU_UTF8_Encoding_Check_HTTP.lua]: @dboltovskyi, done in b42b73aa6e9ec96d8c8743ab9d11b80f8a8c7817

**Comment by dboltovskyi** [comment]
@istoimenova, Script is correct, but currently it won't verify the req-t due to the issue described in "Population of user-friendly messages for PROPRIETARY and HTTP flows" clarification

**Comment by istoimenova** [comment]
@dboltovskyi, thank you for clarification. Please check update in fef767e6960f6cf066e0e96668cb8c8e40d7a92e

**Comment by istoimenova** [comment]
@dboltovskyi, script is update with TODO part. For now will be excluded from test execution. See https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1501

**Pull request merged**